java/configuration/src/main/java/sleeper/configuration/properties/instance/CompactionProperty.java

```diff
@@ -40,6 +40,11 @@ public interface CompactionProperty {
                     "This should be less than the value of sleeper.compaction.queue.visibility.timeout.seconds.")
             .defaultValue("300")
             .propertyGroup(InstancePropertyGroup.COMPACTION).build();
+    UserDefinedInstanceProperty COMPACTION_JOB_VISIBILITY_TIMEOUT_IN_SECONDS = Index.propertyBuilder("sleeper.compaction.job.visibility.timeout.seconds")
+            .description("The delay in seconds until a failed compaction job becomes visible on the compaction job queue and " +
+                    "can be processed again.")
```

Comment:
At one point we were considering using a timeout before the job first becomes visible on the queue. Maybe we should make the property name specific that it's when the job failed?